packages/webdriver/src/index.ts

```diff
@@ -22,8 +22,8 @@ export default class WebDriver {
     ): Promise<Client> {
         const params = validateConfig(DEFAULTS, options)
 
-        if (!options.logLevels || !options.logLevels.webdriver) {
-            logger.setLevel('webdriver', params.logLevel!)
+        if (params.logLevel && (!options.logLevels || !options.logLevels.webdriver)) {
```

Comment:
as in [devtools](https://github.com/webdriverio/webdriverio/blob/main/packages/devtools/src/index.ts#L42)